### DISCUSSION TITLE
[BUG] How to turn off labels inside an article?

### DISCUSSION THREAD

**Discussion by igor17721** [FreshRSS/FreshRSS#5881] [Q&A]
Which part of a code I need to remove in order to delete this tag window? Please note, it's not the same tag which can be turned on/off with tick marks in Settings -> Configuration -> Display (url.com/i/?c=configure&a=display)

![Untitled1](https://github.com/FreshRSS/FreshRSS/assets/108469300/35d9ef23-a6a9-47ae-abba-296a1629ff96)

**Comment by Alkarex** [comment]
You could do that, but it's probably a lot easier in settings → display → disable article tags. ;-)

![image](https://github.com/FreshRSS/FreshRSS/assets/202757/a7f62adb-f2de-4046-a37c-046fc87cf74c)

_Originally posted by @Frenzie in https://github.com/FreshRSS/FreshRSS/issues/5869#issuecomment-1811203724_

**Reply by igor17721** [reply]
The setting above doesn't turn off label that I mentioned in my post, because there're two type of tags and that option disables middle tags (I don't know how they're called). You can check it yourself.

**Reply by Alkarex** [reply]
Please try again our edge branch. There is a new option https://github.com/FreshRSS/FreshRSS/pull/5884

**Comment by igor17721** [comment]
The setting above doesn't turn off label that I mentioned in my post, because there're two type of tags and that option disables middle tags (I don't know how they're called). You can check it yourself.

**Reply by Alkarex** [reply]
Please try again our edge branch. There is a new option https://github.com/FreshRSS/FreshRSS/pull/5884
